linode/account/framework_datasource.go

```diff
@@ -0,0 +1,110 @@
+package account
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/linode/linodego"
+	"github.com/linode/terraform-provider-linode/linode/helper"
+)
+
+func NewDatasource() datasource.DataSource {
+	return &Datasource{}
+}
+
+type Datasource struct {
+	client *linodego.Client
+}
+
+func (data *DatasourceModel) parseAccount(account *linodego.Account) {
+	data.Email = types.StringValue(account.Email)
+	data.FirstName = types.StringValue(account.FirstName)
+	data.LastName = types.StringValue(account.LastName)
+	data.Company = types.StringValue(account.Company)
+	data.Address1 = types.StringValue(account.Address1)
+	data.Address2 = types.StringValue(account.Address2)
+	data.Phone = types.StringValue(account.Phone)
+	data.City = types.StringValue(account.City)
+	data.State = types.StringValue(account.State)
+	data.Country = types.StringValue(account.Country)
+	data.Zip = types.StringValue(account.Zip)
+	data.Balance = types.Float64Value(float64(account.Balance))
+	data.ID = types.StringValue(account.Email)
+}
+
+func (d *Datasource) Configure(
```

Comment:
Is `Configure` necessary here? I know it's in the token migration example, but I'm only able to find it in the [provider migration](https://developer.hashicorp.com/terraform/plugin/framework/migrating/providers#framework-1) overview, instead of any data source or resource migration instructions.